Gemfile.lock

```diff
@@ -67,7 +67,7 @@ GEM
     pry-byebug (3.9.0)
       byebug (~> 11.0)
       pry (~> 0.13.0)
-    racc (1.6.0)
+    racc (1.5.2)
```

Comment:
Is downgrading `racc` an intended change?

Reply:
I don't know where did this come from - reverted!